Makefile

```diff
@@ -51,10 +51,10 @@ dev-dynamic-mem: dev-dynamic
 # Creates a Docker image by adding the compiled linux/amd64 binary found in ./bin.
 # The resulting image is tagged "vault:dev".
 docker-dev: prep
-	docker build --build-arg VERSION=$(GO_VERSION_MIN) --build-arg BUILD_TAGS="$(BUILD_TAGS)" -f scripts/docker/Dockerfile -t vault:dev .
+	docker build --build-arg PRODUCT_VERSION=$(GO_VERSION_MIN) --build-arg BUILD_TAGS="$(BUILD_TAGS)" -f scripts/docker/Dockerfile -t vault:dev .
 
 docker-dev-ui: prep
-	docker build --build-arg VERSION=$(GO_VERSION_MIN) --build-arg BUILD_TAGS="$(BUILD_TAGS)" -f scripts/docker/Dockerfile.ui -t vault:dev-ui .
+	docker build --build-arg PRODUCT_VERSION=$(GO_VERSION_MIN) --build-arg BUILD_TAGS="$(BUILD_TAGS)" -f scripts/docker/Dockerfile.ui -t vault:dev-ui .
```

Comment:
This and the target above seem to be referencing `./scripts/docker/Dockerfile[.ui]` so they shouldn't need updating based on the change to the `./Dockerfile` as far as I can see. The `VERSION` in these Dockerfiles is the Go version, as opposed to the product version. (This is why I always use `PRODUCT_VERSION` for the product version, not just `VERSION` as it helps to disambiguate all the different things with versions we're dealing with :) )

Reply:
Good catch! Just changed it back!